client/app/components/packages/proposed-actions.hbs

```diff
@@ -0,0 +1,120 @@
+<div>
+  <h5 class="small-margin-bottom">
+    Listed below are actions proposed in this project
+  </h5>
+</div>
+{{#each @rwcdsForm.affectedZoningResolutions as |landUseAction|}}
+  <fieldset class="fieldset relative">
+    <legend>
+      <strong>
+        <span class="label">
+          {{~landUseAction.dcpZoningresolutiontype~}}
+          {{#if (lookup-action-type landUseAction.dcpZoningresolutiontype)}}
+            <LabsUi::IconTooltip
+              @tip={{lookup-action-type landUseAction.dcpZoningresolutiontype}}
+              @icon="info-circle"
+              @side="bottom"
+              @transform="shrink-4 up-2"
+              @fixedWidth={{true}}
+            />
+          {{/if}}
+        </span>
+        {{#if (in-array this.actionsWithSectionNumber landUseAction.dcpZoningresolutiontype)}}
+          {{landUseAction.dcpZrsectionnumber}}
+        {{/if}}
+        {{#if (and (eq landUseAction.dcpZoningresolutiontype 'ZR') (eq landUseAction.dcpZrsectionnumber 'Appendix F'))}}
+          {{landUseAction.dcpZrsectionnumber}}
+        {{/if}}
+      </strong>
+    </legend>
+    {{#if (in-array this.actionsWithModifiedZrSectionNumberQuestion landUseAction.dcpZoningresolutiontype)}}
+      <label class="action-form-label">
+        Which sections of the Zoning Resolution does this modify?
+        <small class="help-text display-block">
+          Provide the Zoning Resolution section number(s). Ex. ZR Sec. 42-10 and 43-17
+        </small>
+          <TextInput
+            @property={{landUseAction.dcpModifiedzrsectionnumber}}
+            @propertyName="dcpModifiedzrsectionnumber"
+            @showCounter={{false}}
+            data-test-input="{{landUseAction.dcpZoningresolutiontype}}"
+          />
+      </label>
+    {{else if (and (eq landUseAction.dcpZoningresolutiontype "ZR") (eq landUseAction.dcpZrsectionnumber "Appendix F"))}}
+      This action {{if @rwcdsForm.dcpIncludezoningtextamendment "includes" "does not include"}} MIH. If this is a mistake, please contact City Planning. 
+    {{else}}
+      <span class="help-text">No additional information required for this action</span>
+    {{/if}}
+
+  </fieldset>
+{{/each}}
+<label>
+  <strong>Describe the purpose and need for all the proposed actions.</strong>
+  <TextArea
+    @property={{@rwcdsForm.dcpPurposeandneedfortheproposedaction}}
+    @propertyName="dcpPurposeandneedfortheproposedaction"
+    data-test-input="dcpPurposeandneedfortheproposedaction"
+
+  />
+</label>
+<legend>
+  <strong>Does the applicant plan to develop a 100% affordable housing development?</strong>
+</legend>
+<RadioButton
```

Comment:
Another stylistic option for generating the radio buttons is to use the hash pattern we've used in the pas-form:
```
{{#each (array
          (hash
            code=true
            label="Yes"
          (hash
            code=false
            label="No"
        ) as |radio|}}
          <RadioButton
            @value={{radio.code}}
            @groupValue={{@rwcdsForm.dcpIsplannigondevelopingaffordablehousing}
            @changed={{fn (mut @rwcdsForm.dcpIsplannigondevelopingaffordablehousing) false}}
            data-test-radio-no="dcpIsplannigondevelopingaffordablehousing"
          >
            {{radio.label}}
          </RadioButton>
        {{/each}}
```

Reply:
Note to self/team: Wondering if in future PRs we can create a "SaveableProposedActions" component just like SaveableForm. The component yields radio buttons that triggers validations on Propose Actions changesets on radio button click.

Reply:
In terms of validations, since this is a child model of rwcds form, we'll probably just replicate what we did with applicants on the pas form.

Reply:
As for radio buttons -- will do!